### PR TITLE
[KOGITO-3418] - Update envs field in kogito CRDs to match with native…

### PR DESCRIPTION
**Pull request by Kaitou786**
… k8s objects

See: https://issues.redhat.com/browse/KOGITO-3418

Changing to match the native k8s objects...See: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/ 
Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
Despite the persitence errors, I'm seeing a problem with the Explainability service:

```
Step Explainability result for execution "LoanEligibility" in the Trusty service within 3 minutes with saliences:: Timeout waiting for Get decisions by execution ID bdbb9f9d-5266-4d8c-854f-26abdfb98822
```

@sutaakar maybe infra resource?

File: pkg/apis/app/v1alpha1/kogitoservices_types.go [hunk 1]
```diff
@@ -134,7 +134,7 @@ type KogitoServiceSpec struct {
 	// +listType=atomic
 	// Environment variables to be added to the runtime container. Keys must be a C_IDENTIFIER.
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
-	Envs []corev1.EnvVar `json:"envs,omitempty"`
+	Envs []corev1.EnvVar `json:"env,omitempty"`
```
Comment (ricardozanini) [inline pkg/apis/app/v1alpha1/kogitoservices_types.go:137]: Why not changing the variable as well?

**Review by ricardozanini — COMMENTED** [review]
Just one small question regarding the renaming of the internal attribute as well.
Despite my concern about being an array, thus a plural, I'd say that we should follow the API and change the internal variables as well.

File: pkg/apis/app/v1alpha1/kogitoservices_types.go (hunk 1 above)
Comment (Kaitou786) [reply pkg/apis/app/v1alpha1/kogitoservices_types.go:137]: Okay, will change the variable name as well :)

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
Let's wait for #587 to be merged before running the CI

**Comment by sutaakar** [comment]
old Jenkinsfile was applied, I guess the PR needs to be rebased to apply correct Jenkinsfile

**Review by vaibhavjainwiz — APPROVED** [review]
LGTM

**Pull request merged**